File: emulinker/src/main/java/org/emulinker/kaillera/master/StatsCollector.java
package org.emulinker.kaillera.master;

import java.util.List;
import org.emulinker.kaillera.model.KailleraGame;
import org.emulinker.kaillera.model.KailleraServer;

public interface StatsCollector {
  public void gameStarted(KailleraServer server, KailleraGame game);

  public List<String> getStartedGamesList();

  public void clearStartedGamesList();
}

File: emulinker/src/main/java/org/emulinker/kaillera/master/StatsCollector.kt
package org.emulinker.kaillera.master

import org.emulinker.kaillera.model.KailleraGame
import org.emulinker.kaillera.model.KailleraServer

interface StatsCollector {
  fun markGameAsStarted(server: KailleraServer, game: KailleraGame)

  fun getStartedGamesList(): MutableList<String>

  fun clearStartedGamesList()
}
